### PR TITLE
Moar fixes

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: spot-client/src/spot-remote/ui/views/help.js [hunk 1]
```diff
@@ -52,8 +52,8 @@ class Help extends React.Component {
                         <div className = 'help-message'>
                             This app is a remote controller for a conference room.
                             <div>
-                                Open { Help._getSpotTvUrl() } in a browser to setup your conference room
-                                and obtain a share key.
+                                Open { Help._getSpotTvUrl() } in a desktop browser in the conference room
```
Comment (virtuacoplenny) [inline spot-client/src/spot-remote/ui/views/help.js]: Should we specify chrome browser?

Comment (saghul) [reply spot-client/src/spot-remote/ui/views/help.js]: Indeed!

**Comment by saghul** [comment]
@virtuacoplenny Updated, PTAL.

File: spot-client/src/spot-remote/ui/views/help.js [hunk 2]
```diff
@@ -52,8 +52,8 @@ class Help extends React.Component {
                         <div className = 'help-message'>
                             This app is a remote controller for a conference room.
                             <div>
-                                Open { Help._getSpotTvUrl() } in a browser to setup your conference room
-                                and obtain a share key.
+                                Open { Help._getSpotTvUrl() } in a desktop Google Chrome browser in the conference room
+                                to set it up and obtain a share key. Next enter the 6-digit share key in this app.
```
Comment (virtuacoplenny) [inline spot-client/src/spot-remote/ui/views/help.js]: Nit: 6-character. Let's reduce the particles of dust that can turn into a rejection pearl.

**Pull request merged**